spec/player_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../lib/player.rb'
+require_relative '../lib/board.rb'
+
+describe Player do
+  describe '.player_turn' do
+    it 'should return true if it is an instance of a the Player class' do
+      player_one = Player.new('sam', 'X')
+      player_two = Player.new('chris', 'O')
+      curr_board = Board.new
+      test_player = Player.player_turn(player_one, player_two, curr_board)
+      expect(test_player).to be_a(Player)
+    end
+  end
```

Comment:
See previous comments.

Also, I think this test is redundant when `player_turn` is called, it will always return a `Player` class, why not test which player class it will return instead? 🤔
This way we can test what player will be returned after called `player_turn` 2, 3 or even 4 times? 
Using this approach, we are testing for both negative and positive cases.